src/server/routes/changePassword.ts

```diff
@@ -7,13 +7,13 @@ import {
   validate as validateToken,
   change as changePassword,
 } from '@/server/lib/idapi/changePassword';
-import { ChangePasswordErrors } from '@/shared/model/Errors';
 import { ResponseWithLocals } from '@/server/models/Express';
 import { trackMetric } from '@/server/lib/AWS';
 import { Metrics } from '@/server/models/Metrics';
 import { removeNoCache } from '@/server/lib/middleware/cache';
 import { PageTitle } from '@/shared/model/PageTitle';
 import { setIDAPICookies } from '@/server/lib/setIDAPICookies';
+import { passwordValidation } from '@/client/components/PasswordValidation';
```

Comment:
this is calling the code from the client folder, since it's shared between the server and client, i suggest moving this to the shared folder

Reply:
in fact, keep the component itself in the client, and move over the shared functionality to the shared lib folder?

Reply:
mostly just to keep a consistent style on which code is being used where